src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,19 @@
+.catalog {
+  position: relative;
+  width: $catalogWidth;
```

Comment:
It seems that the variables $catalogWidth and $catalogHeight are not defined in this file. Make sure to import the file containing these variables or define them in this file.